src/utils/deployPeertube.ts

```diff
@@ -42,156 +42,43 @@ export default async function deployPeertube(data: VM, profile: IProfile) {
 
   // Make sure the name is valid
   name = await getSuitableGateway(client, name);
-  // console.log(name);
 
   // Gateway node Id and domain
-  const gwNodeId = 8;
+  const gwNodeId = 7;
```

Comment:
Why is this hard coded?

Reply:
please update to use https://github.com/threefoldtech/grid_weblets/issues/221#issuecomment-993889718